src/dom.js

```diff
@@ -45,11 +45,16 @@ const registerDom = (namespace, registry = {}) => {
 			}
 
 			// observe store usage and process any new effects that were called when building the component
-			const tagResult = observeTag(() => processEffects(populatedTagFunction))
+			const processEffectsAndBuildTagResult = () => processEffects(populatedTagFunction)
+			const tagResult = observeTag(processEffectsAndBuildTagResult)
```

Comment:
This is only because it makes it easier to inspect on the performance tab (otherwise it just appears as `anonymous function`)